### ISSUE TITLE
bootstrap is not loaded first, dataProviders are called before bootstrap

### ISSUE THREAD

**Issue by jordanqq** [Codeception/Codeception#862]
lets say i have my codeception.yml test path to "test" and look for bootstrap in "framework/_boostrap.php". and i have one test suite called "unit".

my files will be laid out like this:
```
test/framework/_bootstrap.php
test/unit/framework/_bootstrap.php
test/unit/..../.../mytestfile.php
```
Codeception will load them in this order:
```
1.  test/framework/_bootstrap.php
2. test/unit/..../.../mytestfile.php
3. test/unit/framework/_bootstrap.php
```

Additionally, if I declare any dataProvider methods in the test file, mytestfile.php, those methods will be called before test/unit/framework/_bootstrap.php is loaded.

**Comment by coviex** [comment]
*unit/_bootstrap.php*
```php
require_once __DIR__ . '/_lib/MyTestCase.php';
```
>$ php ./bin/codecept run unit
>...
>Codeception PHP Testing Framework v2.0.11
>...
>Fatal error: Class 'MyTestCase' not found in /home/user/www/project/tests/unit/topic/ExampleTest.php

Importing file in root bootstrap and all other tests work ok.

**Comment by DavertMik** [comment]
I hope this was fixed in #2872
